### DISCUSSION TITLE
How to update the workflow?

### DISCUSSION THREAD

**Discussion by pawamoy** [best-of-lists/best-of#155] [Q&A]
Hello!

What's the recommended way to update my best-of list with the latest changes from upstream? I get deprecation warnings in my job logs (example: https://github.com/pawamoy/best-of-mkdocs/actions/runs/4822964090).

> Node.js 12 actions are deprecated.

> The `set-output` command is deprecated and will be disabled soon.

**Comment by oprypin** [comment]
>  Node.js 12 actions are deprecated.

That's coming from `actions/checkout@v2`, can be just changed to `v3`

> The `set-output` command is deprecated and will be disabled soon.

That's coming from `actions/create-release@v1`, for which there is no upgrade, it's archived...

I found this by clicking "View raw logs", then the warnings are attached to its relevant action call.

**Reply by pawamoy** [reply]
I see, thanks. I've used https://github.com/softprops/action-gh-release in another project indeed.
